zirgen/Dialect/Zll/IR/Field.cpp

```diff
@@ -171,6 +171,15 @@ ExtensionField::FieldResult ExtensionField::BitAnd(ExtensionField::FieldArg a,
   return c;
 }
 
+ExtensionField::FieldResult ExtensionField::Mod(ExtensionField::FieldArg a,
+                                                ExtensionField::FieldArg b) const {
+  ExtensionField::FieldResult c(degree);
+  for (size_t i = 0; i < degree; i++) {
+    c[i] = a[i] % b[i];
+  }
+  return c;
+}
+
```

Comment:
In C++, `x % 0` is undefined, so I think this would often give undefined behavior. Do we currently use this with non-basefield elements?